frontend/src/concepts/pipelines/content/configurePipelinesServer/ObjectStorageSection.tsx

```diff
@@ -66,15 +59,6 @@ export const ObjectStorageSection = ({
                   onChange={(_, value) => onChange(field.key, value)}
                 />
               </InputGroupItem>
-              {loaded && !!dataConnections.length && (
-                <Tooltip content="Populate the form with credentials from your selected data connection">
-                  <PipelineDropdown
-                    config={config}
-                    setConfig={setConfig}
-                    dataConnections={dataConnections}
-                  />
-                </Tooltip>
-              )}
```

Comment:
Woah, nope, undo. Definitely need this, it was a whole heck of an effort to get it in the first place 🙂 Basically we need to be able to clone the data from S3 Connection instances into this modal... this is how we do it.